decode/radio_frame.js

```diff
@@ -1,45 +1,43 @@
 var EthernetAddr = require('./ethernet_addr');
 var LogicalLinkControl = require('./llc_packet');
-
+var RadioBeaconFrame = require('./radio_beacon_frame');
 function RadioFrame() {
```

Comment:
Putting all of the potential members in the constructor helps V8 not change the hidden class, but admittedly in this case it's not much of a perf win.

The bigger win is for humans reading the code to better understand what these objects are likely to contain. It also makes it more obvious when you write tests what fields to validate. Also, all of the other decoders work this way, so I'd like to keep them consistent.